book/10-git-internals/sections/transfer-protocols.asc

```diff
@@ -1,57 +1,58 @@
-This protocol is called ``dumb'' because it requires no Git-specific code on the server side during the transport process; the fetch process is a series of HTTP `GET` requests, where the client can assume the layout of the Git repository on the server.
+Wenn Sie ein Repository einrichten, das schreibgeschützt über HTTP bereitgestellt wird, wird wahrscheinlich das dumme Protokoll verwendet.
+Dieses Protokoll wird als „dumm“ bezeichnet, da es während des Transportvorgangs keinen Git-spezifischen Code auf der Serverseite erfordert. Der Abrufprozess besteht aus einer Reihe von HTTP `GET` Abfragen, bei denen der Client das Layout des Git-Repositorys auf dem Server übernehmen kann.
 
 [NOTE]
 ====
-The dumb protocol is fairly rarely used these days.
-It's difficult to secure or make private, so most Git hosts (both cloud-based and on-premises) will refuse to use it.
-It's generally advised to use the smart protocol, which we describe a bit further on.
+Das dumme Protokoll wird heutzutage ziemlich selten verwendet.
+Es ist schwierig, es sicher oder privat einzurichten, daher lehnen die meisten Git-Hosts (sowohl cloudbasiert als auch on-premise) die Verwendung ab.
+Es wird generell empfohlen, das smarte Protokoll zu verwenden, welches wir weiter unten beschreiben.
 ====
 
-Let's follow the `http-fetch` process for the simplegit library:
+Folgen wir dem `http-fetch`-Prozess für die simplegit-Bibliothek:
 
 [source,console]
 ----
 $ git clone http://server/simplegit-progit.git
 ----
 
-The first thing this command does is pull down the `info/refs` file.
-This file is written by the `update-server-info` command, which is why you need to enable that as a `post-receive` hook in order for the HTTP transport to work properly:
+Das erste, was dieser Befehl tut, ist das pullen der Datei `info/refs`.
+Diese Datei wird mit dem Befehl `update-server-info` geschrieben. Aus diesem Grund müssen Sie diesen als `post-receive`-Hook aktivieren, damit der HTTP-Transport ordnungsgemäß funktioniert:
 
 [source]
 ----
 => GET info/refs
 ca82a6dff817ec66f44342007202690a93763949     refs/heads/master
 ----
 
-Now you have a list of the remote references and SHA-1s.
-Next, you look for what the HEAD reference is so you know what to check out when you're finished:
+Jetzt haben Sie eine Liste der remote Referenzen und der SHA-1s.
+Als Nächstes suchen Sie nach der HEAD-Referenz, damit Sie wissen, was Sie abschließend überprüfen müssen:
 
 [source]
 ----
 => GET HEAD
 ref: refs/heads/master
 ----
 
-You need to check out the `master` branch when you've completed the process.
-At this point, you're ready to start the walking process.
-Because your starting point is the `ca82a6` commit object you saw in the `info/refs` file, you start by fetching that:
+Sie müssen den 'master'-Branch auschecken, wenn Sie den Vorgang abgeschlossen haben.
+//watch - was ist ein 'walking prozess'?
+Jetzt können Sie mit dem „Walking“-Prozess beginnen.
```

Comment:
Könnte damit der "fortschreitende" Prozess gemeint sein?
In den Texten sind öfter Analogien zum "Laufen" zu finden, wenn z.B. neue Commits eingebunden werden sollen und die Referenz dann verschoben wird . 
Ich habe das im Merging-Kapitel auch mal mit "vorspulen" übersetzt

Reply:
Hm, was soll denn ein fortschreitender Prozess hier konkret sein?
Vorspulen habe ich (wenn überhaupt) für 'fast-forward' genutzt. Darum scheint es hier jedoch nicht zu gehen.

Reply:
Ich meine im Sinn von "aktiver", "laufender" Prozess, der nicht statisch, sondern in Bewegung/Entwicklung ist.

Reply:
Verstehe. Wie soll ich es am besten übersetzen? 
- 'Fortschreitender Prozess'
- oder es einfach bei 'Prozess' belassen

Reply:
Ich glaube ich würde folgendes nehmen:
```suggestion
Jetzt können Sie mit dem laufenden Prozess beginnen.
```